### ISSUE TITLE
Border: Class uses `string` for `title`, not `ustring`

### ISSUE THREAD

**Issue by tig** [gui-cs/Terminal.Gui#1827]
<img width="534" alt="image" src="https://user-images.githubusercontent.com/585482/174343834-a35978d0-a3c1-4c93-8f7f-d08306e762fa.png">

In addition, strings (both `string` and `ustring`) should be initizlied to `.Empty` and not `null`. 

Fixing this may be a breaking change.

**Comment by BDisp** [comment]
But you are having some issue with this? If so, then you can do `ustring.Make(title)` that return a ustring.

**Comment by BDisp** [comment]
Why not added it in the `NStack`? Do you I do that?

**Comment by tig** [comment]
> Why not adding it in the `NStack`? Do you want I do that?

I believe:

1) Classes should always explicitly initialize members (so `private type foo` is BAD and `private type foo = bar` is good. 
2) strings (both `string` and `ustring`) should always be initialized to `.Empty` instead of `null`.

Changing `Nstack` to do this encourages bad developer behavior. So no, I don't  think you should do this to NStack.

**Comment by BDisp** [comment]
> In addition, strings (both `string` and `ustring`) should be initizlied to `.Empty` and not `null`.

we cant initialize with a default string.Empty, see the error:

![imagem](https://user-images.githubusercontent.com/13117724/181924761-59b234b0-47d6-4187-b8fb-98c9440e1346.png)

**Comment by tznind** [comment]
> we cant initialize with a default string.Empty, see the error:

You could do

```csharp
public TopLevelContainer (Border border, string title = null)
{
    Initialize(Rect.Empty,border, title ?? string.Empty);
}
```

**Comment by BDisp** [comment]
Does this issue can be closed per #1862?
